mbuild/formats/lammpsdata.py

```diff
@@ -113,8 +121,31 @@ def write_lammpsdata(structure, filename, atom_style='full',
     unique_types = list(set(types))
     unique_types.sort(key=natural_sort)
 
-    charges = [atom.charge for atom in structure.atoms]
+    charges = np.array([atom.charge for atom in structure.atoms])
+
+    # Convert coordinates to LJ units
+    if unit_style == 'lj':
+        # Get sigma, mass, and epsilon conversions by finding maximum of each
+        sigma_conversion_factor = np.max([atom.sigma for atom in structure.atoms])
```

Comment:
I've never used lj units for lammps, but would some kwargs to specify the conversion factors be useful in addition to the "auto detection" for the conversion factors? We do this for hoomd sometimes